test_framework/terraform/harvester/ubuntu/main.tf

```diff
@@ -0,0 +1,146 @@
+terraform {
+  required_providers {
+    rancher2 = {
+      source  = "rancher/rancher2"
+      version = "3.0.0-rc2"
```

Comment:
Why are we using `3.0.0-rc2`, the latest is [v4.1.0](https://github.com/rancher/terraform-provider-rancher2/releases/tag/v4.1.0)?

Reply:
Refined. Thank you!